matRadGUI.m

```diff
@@ -3670,19 +3693,32 @@ function edit_windowRange_Callback(hObject, eventdata, handles)
 %        str2double(get(hObject,'String')) returns contents of edit_windowRange as a double
 
 selectionIndex = get(handles.popupmenu_chooseColorData,'Value');
+vRange = str2num(get(hObject,'String'));
+% matlab adds a zero in the beginning when text field is changed
+if numel(vRange) == 3
+    vRange = vRange(vRange~=0);
+end
 
 switch selectionIndex 
     case 2
-        handles.ctWindow = str2num(get(hObject,'String'));
+        handles.ctWindow   = vRange;
     case 3
-        handles.doseWindow = str2num(get(hObject,'String'));
+        handles.doseWindow = vRange;
     otherwise
 end
 
 handles.cBarChanged = true;
 
-guidata(hObject,handles);
-UpdatePlot(handles);
+if vRange(1) < vRange(2)    
```

Comment:
I am very confused at this point. Looks like high end spaghetti to me. Sorry.

Why is there handles.vRangeDose and handles.doseWindow?

Why are lines 3712 and following also executed for case 2, i.e. when modifying ct?